Scripts/script-ReadFile.yml

```
@@ -36,7 +39,11 @@ args:
   required: false
   default: false
   description: Maximal file size to load, in bytes. Default is 1MB.
+- name: decode_type
```

Let's change the name to: encoding
```suggestion
- name: encoding
```